rest-server/test/jobSubmission.js

```diff
@@ -154,7 +215,7 @@ describe('Submit job: POST /api/v1/jobs', () => {
     global.chai.request(global.server)
       .post('/api/v1/jobs')
       .set('Authorization', 'Bearer ' + validToken)
-      .send(JSON.parse(global.mustache.render(global.jobConfigTemplate, {'jobName': 'job1'})))
+      .send(JSON.parse(global.mustache.render(global.jobConfigTemplate, { 'jobName': 'job1' })))
```

Comment:
`object-curly-spacing` lint error, e.g., in [build #1125](https://travis-ci.org/Microsoft/pai/jobs/369736701)